extensions/flight-sql/src/main/java/io/deephaven/server/flightsql/FlightSqlErrorHelper.java

```diff
@@ -0,0 +1,25 @@
+//
+// Copyright (c) 2016-2024 Deephaven Data Labs and Patent Pending
+//
+package io.deephaven.server.flightsql;
+
+import io.grpc.Status;
+import io.grpc.StatusRuntimeException;
+
+final class FlightSqlErrorHelper {
```

Comment:
This is targeted at the gRPC API as opposed to the proto API; not that this precludes us from having them added to `io.deephaven.proto.util.Exceptions`. And this _is_ somewhat FlightSQL specific, at least b/c it is prefixing with `"Flight SQL: ". As such, it makes sense for FlightSqlErrorHelper to still exist. I think it would make sense to migrate the basic structure of these methods to somewhere more generally available as soon as somebody else wants to use the same pattern, but for now probably minimize churn and leave as is?

Reply:
Technically not necessary, but I do find it makes the error messages a bit more scoped without needing to traverse a stack trace. If we think the redundancy is harmful, will remove.